lib/Ama/Model/Votes.pm

```diff
@@ -30,4 +37,18 @@ sub uncast {
   $@ ? { error => $@ } : $results;
 }
 
+sub _autoRemove {
```

Comment:
I reckon you chose autoremove because delete is a reserved keyword?  What about just remove?  Why "auto"?  It almost feels redundant.